### ISSUE TITLE
Gamma settings?

### ISSUE THREAD

**Issue by tinkerer-red** [terabyte25/tes3mp-android#8]
Is there a setting for gamma in the configuration files? my phone even at max brightness in the middle of the night it's still hard to see some of the caves/dungeons in the game even with a torch.

**Comment by BrittonGage** [comment]
In the play store version of OpenMW there isn't but there are likely
several third party apps as well as the night eye spell you could use.

**Comment by tinkerer-red** [comment]
Yeah I've been stacking up on torches and night eye although even with both on at once i cant see shit in caves. might be best to make use of the soultrap glitch, so i can have stack-able infinite night eyes.

**Comment by tinkerer-red** [comment]
So I found an app which helps with this issue. It's called color calibrator. Just turn up the white/grey and all your blacks will be brighter with out raising your brightness. With it I can explore caves while I'm outside in daylight, if I set it all the way up with brightness I can play with the sun reflecting in my eyes. Not that I would normally do that, but I was able to.
